### ISSUE TITLE
Docs request: upstream config example

### ISSUE THREAD

**Issue by miles-** [Kong/kubernetes-ingress-controller#3]
First of all, thank you for much for this project!

I am trying to customize one of the upstreams that the kong ingress controller created. Specifically, I want to add `hash_on` to one of the upstreams per the admin API [docs](https://getkong.org/docs/0.13.x/admin-api/#add-upstream).

The README notes:
> Instead it uses the Endpoints API to bypass kube-proxy to allow Kong features like session affinity and custom load balancing algorithms.

Is there an example of how I can add session affinity to an ingress resource?

**Comment by aledbf** [comment]
> Is there an example of how I can add session affinity to an ingress resource?

Not yet. We need to add a new Kubernetes type to allow customization of the upstream and health checks. This is what we are working right now.

**Comment by miles-** [comment]
@aledbf Understood, thanks for the quick reply.

**Comment by aledbf** [comment]
> is the same thing true for configuring routes?

Yes. This applies to all the configurations you can change in a route, service and upstream

**Comment by miles-** [comment]
@aledbf Got it, thanks.
